optax/losses/_self_supervised_test.py

```diff
@@ -53,5 +53,41 @@ def test_batched(self):
     )
 
 
+class TripletMarginLossTest(chex.TestCase):
+
+  def setUp(self):
```

Comment:
Avoid using numerical values as expected returns.
They may fail depending on the backend for example.
You may consider simple test cases with a "handmade" function (see e.g. the lbfgs tests). You can check for specific inputs (like zeros or ones).

You may also add a test for some specific behaviors (like using swap here).

Also you should test this function under jit/vmap etc... (see the chex.all_variant utility in some other tests).